src/applications/vaos/project-cheetah/components/VAFacilityPage/VAFacilityInfoMessage.jsx

```diff
@@ -1,28 +1,30 @@
 import React from 'react';
 import AlertBox from '@department-of-veterans-affairs/component-library/AlertBox';
 import State from '../../../components/State';
-import NewTabAnchor from '../../../components/NewTabAnchor';
 
-export default function VAFacilityInfoMessage({ facility }) {
+export default function VAFacilityInfoMessage({ facility, sortMethod }) {
   return (
-    <AlertBox
-      status="info"
-      headline="We found one VA location for you"
-      level="2"
-    >
+    <div className="vads-u-margin-bottom--4">
       <p>
-        <strong>{facility.name}</strong>
-        <br />
-        {facility.address?.city}, <State state={facility.address?.state} />
+        We found one VA location where you're registered that offer COVID-19
+        vaccine appointments.
       </p>
-      Not all VA locations offer all types of care or support online scheduling.
-      <p>
-        If this location wasn't what you were looking for, you can{' '}
-        <NewTabAnchor href="/find-locations">
-          search for a nearby location
-        </NewTabAnchor>{' '}
-        and call to schedule an appointment.
-      </p>
-    </AlertBox>
+      <strong>{facility.name}</strong>
+      <br />
+      {facility.address?.city}, <State state={facility.address?.state} />
+      <br />
+      {!!facility.legacyVAR[sortMethod] && (
+        <>
+          {facility.legacyVAR[sortMethod]} miles
+          <br />
+        </>
+      )}
+      <AlertBox
```

Comment:
I included the second dose info alert to match the vaccine appointment flow for multiple facilities. I can remove it since it's somewhat redundant for single facilities but it seemed a good warning to potentially include.